src/m365/spo/commands/file/file-move.spec.ts

```diff
@@ -16,18 +18,10 @@ describe(commands.FILE_MOVE, () => {
   let commandInfo: CommandInfo;
 
   const stubAllPostRequests: any = (
-    recycleFile: any = null,
     createCopyJobs: any = null,
     waitForJobResult: any = null
   ) => {
     return sinon.stub(request, 'post').callsFake((opts) => {
-      if ((opts.url as string).indexOf('/recycle()') > -1) {
```

Comment:
Is this test failing if we let it as-is? In theory, we shouldn't have to expose the implementation details to tests. In other words: it should be irrelevant to the test if the command calls another command or executes the request itself as long as the file is deleted, which we can verify by checking for the request. If this test is working, then we can keep it as-is.